source/likely.js

```diff
@@ -2,5 +2,6 @@
 
 var likely = require('./index.js');
 
-window.likely = likely;
 window.addEventListener('load', likely.initiate);
```

Comment:
@andrew--r What about you? I have some time to implement build in two bundles, as we discussed before. You and @iamakulov could just review this on weekends. If you'd like to make it by yourself, than it's okay and I woun't.